plugins/package-managers/swiftpm/src/main/kotlin/SwiftPmModel.kt

```diff
@@ -70,6 +71,23 @@ data class PinV1(
     )
 }
 
+/**
+ * See https://github.com/apple/swift-package-manager/blob/3ef830dddff459e569d6e49c186c3ded33c39bcc/Sources/PackageGraph/PinsStore.swift#L387-L462.
+ */
+@Serializable
+data class PinV2(
+    val identity: String,
+    val state: PinV1.State?,
```

Comment:
Shouldn't this be `(PinV2.)State`? Otherwise that new state class would be unused.